mythril/laser/ethereum/state.py

```diff
@@ -141,8 +141,11 @@ def mem_extend(self, start, size):
         :param start: Start of memory extension
         :param size: Size of memory extension
         """
-        self.memory += [0] * max(0, start + size - self.memory_size)
-
+        if self.memory_size > start + size:
```

Comment:
Why this change?

Reply:
thats seems faster than the older method.

Reply:
Is it much faster? I liked the cleanness of the previous implementation, and if its just a small performance improvement then I'd like to keep it like that

Reply:
I am not sure it is big or small but it is around 20%

Reply:
also this method somehow survived some of the sha3 tests while the former killed my ram, but those tests were meant for gas exhaustion so it might not matter.